src/angular/autocomplete/autocomplete-trigger.ts

```diff
@@ -457,7 +462,6 @@ export class SbbAutocompleteTrigger
   }
 
   /** @docs-private */
-  @HostListener('keydown', ['$event'])
   _handleKeydown(event: TypeRef<KeyboardEvent>): void {
```

Comment:
You can remove the TypeRef wrapper as this is only required when using an event type parameter with decorators. (e.g. `TypeRef<KeyboardEvent> => KeyboardEvent`)